drivers/block/rbd.c

```diff
@@ -5269,7 +5271,17 @@ static int rbd_dev_v2_header_onetime(struct rbd_device *rbd_dev)
 			goto out_err;
 	}
 
-	rbd_init_layout(rbd_dev);
+	ret = rbd_init_layout(rbd_dev);
+#if 0 /* TODO(dis) */
```

Comment:
Nit: remove reminder

Reply:
This one is actually something I wanted to talk about.  I'll bring it up tomorrow.